spid_cie_oidc/provider/tests.py

```diff
@@ -1 +1,45 @@
-# Create your tests here.
+
+from django.http import HttpRequest
+from django.test import TestCase
+from spid_cie_oidc.authority.tests.settings import *
+from spid_cie_oidc.entity.jwtse import create_jws
+from spid_cie_oidc.entity.models import FetchedEntityStatement, TrustChain
+from spid_cie_oidc.entity.tests.settings import *
+from spid_cie_oidc.provider.settings import *
+from spid_cie_oidc.provider.views import authn_request
+
+
+class AuthnRequestTest(TestCase):
+
+    def setUp(self):
+        self.req = HttpRequest()
+
+
+    def test_auth_request(self):
+        jws=create_jws(PAYLOAD,JWK)
+        self.req.GET = {'request':jws}
```

Comment:
here the tests MUST be done with a django http client, as @dezhizhang1985 made here:

https://github.com/peppelinux/spid-cie-oidc-django/blob/da3ec7d5b0bbc0d1d4ade1e56fe58616c3e7a52d/spid_cie_oidc/onboarding/tests/test_01_onboarding.py#L34